modules/home/transformations/filter-plugins.php

```diff
@@ -30,17 +29,61 @@ private function get_installed_plugins(): array {
 		return array_keys( $plugins );
 	}
 
+	private function is_plugin_activated( $plugin_path ) {
+		return $this->wordpress_adapter->is_plugin_active( $plugin_path );
+	}
+
+	private function get_install_plugin_url( $plugin_path ) {
+		$slug = dirname( $plugin_path );
+		$admin_url = $this->wordpress_adapter->self_admin_url( 'update.php?action=install-plugin&plugin=' . $slug );
+		$nonce_url = $this->wordpress_adapter->wp_nonce_url( $admin_url, 'install-plugin_' . $slug );
+		return $this->removeAmpersandFromUrl( $nonce_url );
+	}
+
+	private function get_activate_plugin_url( $plugin_path ) {
+		$nonce_url = $this->wordpress_adapter->wp_nonce_url( 'plugins.php?action=activate&amp;plugin=' . $plugin_path . '&amp;plugin_status=all&amp;paged=1&amp;s', 'activate-plugin_' . $plugin_path );
+		return $this->removeAmpersandFromUrl( $nonce_url );
+	}
+
+	private function removeAmpersandFromUrl( $url ) {
+		return str_replace( '&amp;', '&', $url );
+	}
```

Comment:
@hein-obox Why do we need this? Can we remove it? It's not needed in the admin apps page 🤔

Reply:
@louiswol94 I get this URL on my localhost: http://elementor.local/wp-admin/plugins.php?action=activate&amp;plugin=image-optimization%2Fimage-optimization.php&amp;plugin_status=all&amp;paged=1&amp;s&amp;_wpnonce=d0cb500a28. This URL doesn't work in the browser.

![Screenshot 2024-04-11 at 14 10 48](https://github.com/elementor/elementor/assets/94352322/da054089-04b5-4928-b5ff-6a06a3dff036)

When we echo this in PHP the `$amp` is converted to `$`. 

See e.g. https://stackoverflow.com/questions/50428250/php7-echo-amp-returns-versus-amp.

I am not 100% sure about the reason for this.